src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -222,4 +232,17 @@ private function writeResult(OutputInterface $output)
             '',
         ));
     }
+
+    /*
+     * @internal
+     */
+    public function generateSalt(OutputInterface $output)
+    {
+        $value = base64_encode($this->getContainer()->get('security.secure_random')->nextBytes(16));
+
+        $output->writeln("\n<comment>The salt has been generated: </comment>".$value);
```

Comment:
You should use `sprintf()`  function to remain consistent with the very next statement.